pallets/nbv-storage/src/lib.rs

```diff
@@ -0,0 +1,273 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+pub use pallet::*;
+
+#[cfg(test)]
+mod mock;
+
+#[cfg(test)]
+mod tests;
+
+#[cfg(feature = "runtime-benchmarks")]
+mod benchmarking;
+
+#[frame_support::pallet]
+pub mod pallet {
+use frame_support::pallet_prelude::*;
+	use frame_system::pallet_prelude::*;
+
+	use frame_support::{
+		BoundedVec,
+		transactional,
+		sp_io::hashing::blake2_256,
+	};
+	use scale_info::prelude::boxed::Box;
+
+	//use scale_info::TypeInfo;
+	/// Configure the pallet by specifying the parameters and types on which it depends.
+	#[pallet::config]
+	pub trait Config: frame_system::Config + pallet_identity::Config {
+		/// Because this pallet emits events, it depends on the runtime's definition of an event.
+		type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;
+		#[pallet::constant]
+		type XPubLen: Get<u32>;
+		#[pallet::constant]
+		type PSBTMaxLen: Get<u32>;
+	}
+
+	#[pallet::pallet]
+	#[pallet::generate_store(pub(super) trait Store)]
+	pub struct Pallet<T>(_);
+
+	#[pallet::event]
+	#[pallet::generate_deposit(pub(super) fn deposit_event)]
+	pub enum Event<T: Config> {
+
+		/// Xpub and hash stored. Linked to an account identity
+		XPubStored([u8 ; 32], T::AccountId),
+		/// Removed Xpub previously linked to the account
+		XpubRemoved(T::AccountId),
```

Comment:
This is a tiny complaint, but I think the `P` should be capitalized on both events. Or lowercase, but either way it should be consistent, I think